hasher-matcher-actioner/hmalib/lambdas/api/action_rules_api.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import bottle
+import typing as t
+import sys
+
+from botocore.exceptions import ClientError
+from bottle import response
+from dataclasses import dataclass, field
+from hmalib.common.logging import get_logger
+from .middleware import jsoninator, JSONifiable, DictParseable
+from hmalib.common.config import HMAConfig
+from hmalib.common import config as hmaconfig
+from hmalib.common.evaluator_models import ActionRule
+from hmalib.common.classification_models import Label, ActionLabel
+
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class ActionRulesAPIRequest(DictParseable):
+    action_rule: ActionRule
+
+    @classmethod
+    def from_dict(cls, d: dict) -> "ActionRulesAPIRequest":
+        ar = ActionRule.from_aws(d["action_rule"])
+        logger.info("Deserialized ActionRule: %s", ar)
```

Comment:
nit/ignorable: probably more of a debug level log unless you think we care about this enough every time it happens

Reply:
Sounds good.